### PR TITLE
[FEAT] assert that actions cannot be sent from a destroyed/destroying object

### PR DESCRIPTION
**Pull request by runspired**

### CODE REVIEW

File: packages/ember-glimmer/tests/integration/components/target-action-test.js [hunk 1]
```diff
@@ -259,6 +260,34 @@ moduleFor('Components test: sendAction', class extends RenderingTest {
...
+
+        toString() {
+          return 'component:rip-alley';
+        }
+      })
+    });
+
+    this.render('{{rip-alley}}');
+
+    runDestroy(component);
```
Comment (rwjblue) [inline packages/ember-glimmer/tests/integration/components/target-action-test.js]: I’d vaguely prefer a more natural way to destroy the component, e.g. render it in an `if` and toggle the condition 

What do you think?

Comment (runspired) [reply packages/ember-glimmer/tests/integration/components/target-action-test.js]: @rwjblue didn't think of that, but I agree sticking closer to real world is best. Will update to that. Do we have a helper or do I need to wrap a set in a run?

**Review by stefanpenner — APPROVED** [review]

**Comment by runspired** [comment]
The failure is that browserstack timed out, all else passed. Can we reboot that?

**Comment by rwjblue** [comment]
ya, I kicked that one..

**Review by rwjblue — APPROVED** [review]

**Review by rwjblue — APPROVED** [review]

**Pull request merged**